esi_leap/tests/api/controllers/v1/test_lease.py

```diff
@@ -109,6 +125,121 @@ def test_post_default_resource_type(self, mock_create, mock_cra,
         self.assertEqual(data, request.json)
         self.assertEqual(http_client.CREATED, request.status_int)
 
+    @mock.patch('esi_leap.api.controllers.v1.lease.LeasesController.'
+                '_lease_get_all_authorize_filters')
+    @mock.patch('esi_leap.objects.lease.Lease.get_all')
+    def test_get_nofilters(self, mock_get_all, mock_lgaaf):
+        self.get_json('/leases')
+
+        mock_lgaaf.assert_called_once_with(self.context.to_policy_values(),
+                                           project_id=None,
+                                           start_time=None,
+                                           end_time=None,
+                                           status=None,
+                                           offer_uuid=None,
+                                           view=None,
+                                           owner_id=None,
+                                           resource_type=None,
+                                           resource_uuid=None)
+        mock_get_all.assert_called_once
```

Comment:
assert_called_once(), just my personal preference to make it consistent with other tests. Here and other assert_called_once in this file.

Reply:
Fair point! Fixed.